vault/resource_consul_secret_backend.go

```diff
@@ -296,13 +296,17 @@ func consulSecretsBackendCustomizeDiff(_ context.Context, diff *schema.ResourceD
 	newToken := diff.Get("token").(string)
 	newBootstrap := diff.Get("bootstrap").(bool)
 
-	// If the user sets bootstrap to false but doesn't provide a token, disallow it.
-	if newToken == "" && !newBootstrap {
+	// Disallow the following:
```

Comment:
I think the flow would be a little bit cleaner with a single if/else statement like:

```
if diff.Get("bootstrap").(bool) {
   .... do the validation 
} else {
   .... do the other validation
}
```

Reply:
Please let me know if my latest commit is what you were envisioning.